components/proxy/src/main/java/com/hotels/styx/routing/handlers/RouteRefLookup.java

```diff
@@ -36,21 +36,25 @@
      * A StyxObjectStore based route reference lookup function.
      */
     class RouteDbRefLookup implements RouteRefLookup {
```

Comment:
The indentation below here is incorrect on several lines.

Reply:
This has been fixed.